daemon/cmd/cni.go

```diff
@@ -0,0 +1,354 @@
+package cmd
```

Comment:
This file should have a license/copyright header:

```
// SPDX-License-Identifier: Apache-2.0                                                                                                                                                                                                                                          
// Copyright Authors of Cilium
```

Same for other newly added files. Ideally we'd add a CI check for that in the future :slightly_smiling_face: